vdr/types/common.go

```diff
@@ -136,31 +136,37 @@ type ResolveMetadata struct {
 // CompoundService is a service type that can be used as target for github.com/nuts-foundation/go-did/did/document.go#UnmarshalServiceEndpoint
 type CompoundService map[string]string
 
+// KeyUsage is a bitmask used for specifying for what purposes a key in a DID document can be used (a.k.a. Verification Method relationships).
```

Comment:
KeyUsage is not a mask, but a flag set. When filtered with a logical AND the filter becomes the mask. Flags are unsigned by nature. I would propose `type DocKeyFlags uint` instead.

Reply:
I like the "flags" portion, because `KeyUsage` sounded a bit too much like X509 while it isn't. Renamed to `DIDKeyFlags`.